lp-core-platform/lp-cp-xwiki-implementation/src/main/java/eu/learnpad/core/rest/XWikiRestUtils.java

```diff
@@ -111,6 +111,17 @@ public static boolean createEmptyPage(String wikiName, String spaceName,
 				IOUtils.toInputStream(emptyPageXML));
 	}
 
+	public static String exposeBPMNFromCoreRepository(String basename, String extension){
```

Comment:
Because it may be a list, I guess this function should return a List<String> or Collection<String>.